.ci/e2eTestingHelmDaily.groovy

```diff
@@ -40,7 +40,7 @@ pipeline {
   stages {
     stage('Run Tests') {
       steps {
-        build(job: 'e2e-tests/e2e-testing-mbp/master',
+        build(job: "e2e-tests/e2e-testing-mbp/${ env.JOB_BASE_NAME.startsWith('PR-') ? 'none' : env.JOB_BASE_NAME }",
```

Comment:
Do we still need the if/else condition? JJB is filtering by branch name.

Isn't there a JJB definition to exclude any PR, simplifying this?

Reply:
indeed, IIRC, we use that particular pattern for the upstreamTrigger

Reply:
Should it be?
```suggestion
        build(job: "e2e-tests/e2e-testing-mbp/${ env.JOB_BASE_NAME }",
```